dashboard/src/containers/OperatorNewContainer/OperatorNewContainer.tsx

```diff
@@ -0,0 +1,49 @@
+import { push } from "connected-react-router";
+import { connect } from "react-redux";
+import { Action } from "redux";
+import { ThunkDispatch } from "redux-thunk";
+
+import actions from "../../actions";
+import OperatorNew from "../../components/OperatorNew";
+import { IStoreState } from "../../shared/types";
+
+interface IRouteProps {
+  match: {
+    params: {
+      operator: string;
+    };
+  };
+}
+
+function mapStateToProps(
+  { operators, namespace }: IStoreState,
+  { match: { params } }: IRouteProps,
+) {
+  return {
+    namespace: namespace.current,
+    isFetching: operators.isFetching,
+    operator: operators.operator,
+    error: operators.errors.fetch || operators.errors.create,
```

Comment:
Why not let the component handle `operator.errors` instead? (if there can be multiple concurrently - not sure).

Reply:
both errors cannot happen at the same time. It either fails to gather the operator info (fetch) or once the info is available it fails to create an operator (create). I did this because it's slightly easier to manage in the component logic.